src/plugins/chosen-selectpicker/plugin.js

```diff
@@ -23,7 +23,15 @@ QueryBuilder.define('chosen-selectpicker', function(options) {
     });
 
     this.on('afterCreateRuleOperators', function(e, rule) {
-        rule.$el.find(Selectors.rule_operator).removeClass('form-control').chosen(options);
+        var node = rule.$el.find(Selectors.rule_operator);
+        if (node.css('display') !== 'none') { // avoid creating useless chosen widget
+            rule.$el.find(Selectors.rule_operator).removeClass('form-control').chosen(options);
+        }
+    });
+
+    this.on('afterCreateRuleInput', function(e, rule) {
```

Comment:
Builder plugins are not to be applied on value inputs, this is the purpose of the `filter.plugin` options, please remove this chunk

Reply:
Wow. Completely missed that. Awesome!